### PR TITLE
Build Do you want to upload files for a return

### PR DESCRIPTION
**Pull request by byohannes**
https://eaflood.atlassian.net/browse/SDDSIP-1010
https://eaflood.atlassian.net/browse/SDDSIP-1008
https://eaflood.atlassian.net/browse/SDDSIP-1009

Screenshot 2023-06-23 at 13 24 51

Screenshot 2023-06-23 at 13 03 36

### CODE REVIEW

File: packages/web-service/src/pages/returns/a24/artificial-sett-grid-reference.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ import { gridReferenceValidator } from '../../../pages/common/grid-ref-validator
 import { APIRequests } from '../../../services/api-requests.js'
 
 const { ARTIFICIAL_SETT_GRID_REFERENCE } = ReturnsURIs.A24
+const { LICENCE_CONDITIONS } = ReturnsURIs
 
 const artificialSettGridReference = 'artificial-sett-grid-reference'
 export const validator = payload => gridReferenceValidator(payload, artificialSettGridReference)
```
Comment (GrahamMWillis) [inline packages/web-service/src/pages/returns/a24/artificial-sett-grid-reference.js:11]: Did you reuse the one created for A24?

File: packages/web-service/src/pages/site/common/__tests__/site-map-upload.spec.js [hunk 2]
```diff
@@ -73,16 +72,18 @@ describe('the map of the site showing the mitigations after development page pag
     const request = {
       cache: () => ({
         getData: () => ({
-          fileUpload: { filename: 'site-after-development.shp', path: '/tmp/path' },
```
Comment (GrahamMWillis) [inline packages/web-service/src/pages/site/common/__tests__/site-map-upload.spec.js:76]: I do not believe that *.shape files are accepted --- look at 
packages/web-service/src/pages/common/file-upload/file-upload.js line 8

**Review by GrahamMWillis — COMMENTED** [review]
Yohannes -- please can you just check through my recent changes to the S3 interface to make sure its still Ok? It should be as it should as the new changes should be invisible to the handlers. Did you get a confirmation about the filetypes? Note that *.shape is NOT accepted as commented.

File: packages/web-service/src/pages/returns/a24/artificial-sett-grid-reference.js (hunk 1 above)
Comment (byohannes) [reply packages/web-service/src/pages/returns/a24/artificial-sett-grid-reference.js:11]: licence conditions page is a common one for all species returns

**Pull request merged**